code/modules/recycling/conveyor2.dm

```diff
@@ -64,19 +68,24 @@
 		if(NORTHEAST)
 			forwards = EAST
 			backwards = SOUTH
-		if(NORTHWEST)
-			forwards = SOUTH
-			backwards = WEST
 		if(SOUTHEAST)
 			forwards = NORTH
 			backwards = EAST
 		if(SOUTHWEST)
 			forwards = WEST
 			backwards = NORTH
+		if(NORTHWEST)
+			forwards = SOUTH
+			backwards = WEST

```

Comment:
I'm going to guess this may be related to `setmove` still existing/being used here, but not on TG. Make sure your TG branch is most up to date--that's caught me more than once =p